src/app/components/client/Chart.tsx

```diff
@@ -100,14 +105,32 @@ export const DoughnutChart = (props: Props) => {
     </div>
   );
 
-  const prompt = (
-    <div className={styles.prompt}>
-      <p>{l10n.getString("exposure-chart-returning-user-upgrade-prompt")}</p>
-      <Link href="/redesign/user/welcome">
-        {l10n.getString("exposure-chart-returning-user-upgrade-prompt-cta")}
-      </Link>
-    </div>
-  );
+  const scanInProgressPrompt = <div className={styles.prompt}></div>;
```

Comment:
Doesn't this mean that this is always truthy, for line 124? And what is the value even used for?

Maybe it makes more sense to make it default to `null`, then extract the function body from `getPrompt` to run anyway and assign to this variable, and then just check this variable instead of running the function twice on line 217?

Reply:
Seems like I left this line there unintentionally — updated in [746b6a5](https://github.com/mozilla/blurts-server/pull/3431/commits/746b6a5357a5403d969ecf6aab886d18e197359d).